src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -104,10 +104,23 @@ class ArtworkSidebarExtraLinksContainer extends React.Component<
     window.open(sd.APP_URL + "/consign", "_blank")
   }
 
+  conditionsOfSaleText() {
+    const first = "By placing your bid you agree to Artsy's "
+    if (!this.props.artwork.sale.isBenefit) {
+      const partnerName = this.props.artwork.sale.partner.name
+      const possessivePartnerName = partnerName.endsWith("'s")
+        ? partnerName
+        : partnerName + "'s"
+      return first + "and " + possessivePartnerName + " "
```

Comment:
Small grammar nit but I think that in this case we want to treat Artsy and the partner as a single unit having joint possession over the Conditions of Sale.

For example:

> By placing this bid you agree to Artsy and Seuol Auction's Conditions of Sale.

https://brians.wsu.edu/2016/05/19/joint-possessives/

Reply:
I talked this over with Nicole B, and decided to leave it this way. Effectively they're agreeing the the _respective_ COS of 2 organizations.

Reply:
Gotcha. Makes sense! In that case, do we link to the partner's COS as well?

Reply:
No we don't. We have a blob of text on some auction pages if the partner wants us to do that.